packages/extension/src/chromeServices/background/index.ts

```diff
@@ -45,11 +49,27 @@ import {
   PARAMETER_TRANSACTION_SET_ACCOUNT
 } from '../../constants/parameters';
 import { focusOrCreatePopupWindow } from './utils/focusOrCreatePopupWindow';
+import { Session } from './utils/session';
 
 const sendMessageToTab = <T>(tabId: number | undefined, message: any) => {
   chrome.tabs.sendMessage<T>(tabId ?? 0, message);
 };
 
+async function createOffscreen() {
+  await chrome.offscreen
+    .createDocument({
```

Comment:
We should also close the document when the session expires, and only open it when a session is created